src/modules/sharing/sharing.service.spec.ts

```diff
@@ -491,4 +493,71 @@ describe('Sharing Use Cases', () => {
       );
     });
   });
+
+  describe('get sharing folder size', () => {
+    const owner = newUser();
+    const otherUser = publicUser();
+    const folder = newFolder({ owner });
+
+    it('When user tries to get sharing folder size with a public folder, then it works', async () => {
+      const sharing = newSharing({
+        owner,
+        item: folder,
+        sharedWith: otherUser,
+        sharingType: SharingType.Public,
+      });
+
+      sharingRepository.findOneSharing.mockResolvedValue(sharing);
+      folderUseCases.getFolderSizeByUuid.mockResolvedValue(200);
```

Comment:
Let's extract 200 in a variable so if you change it, you do not have to change it in two different points. Also, I though at first it was a HTTP status code, so the variable naming would help a lot there